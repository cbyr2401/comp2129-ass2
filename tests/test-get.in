GET a
GET b
GET 1
SET a 1 2 3 4
SET b 6 7 8 9
GET a
GET b
SET c
GET c
DEL b
GET b
BYE