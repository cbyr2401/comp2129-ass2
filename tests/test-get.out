> no such key

> no such key

> no such key

> ok

> ok

> [1 2 3 4]

> [6 7 8 9]

> ok

> []

> ok

> no such key

> bye